src/pages/orders/index.tsx

```diff
@@ -0,0 +1,141 @@
+        );
+        setFilterdOrders(result);
+      }, 500),
+    []
+  );
+
+  // handle search bar value
+  const handleSearchChange = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    debouncedChangeHandler(e.target.value || "");
+  }, []);
+
+  // handle order status filter value
+  const handleValueChange = useCallback(
+    (value: FilterType) => {
+      setFilterStatus(value);
+    },
+    [setFilterStatus]
+  );
+
+  if (isFetching && !orders) return null;
+  if (isLoading) return <div>Loading...</div>;
+  if (isError) return <div>Error...</div>;
+  return (
+    <Flex flexDirection="column">
+      <Heading marginY="50px">Orders</Heading>
+      {/* Search bar */}
+      <InputGroup>
+        <InputLeftElement pointerEvents="none">
+          <AiOutlineSearch color="#3C3C3C" />
+        </InputLeftElement>
+        <Input
+          placeholder="Search order ID or user ID ..."
+          _placeholder={{ color: "D9D9D9" }}
+          height="40px"
+          width="200px"
+          fontSize="12px"
+          aria-label="search-input"
+          onChange={handleSearchChange}
+        />
+      </InputGroup>
+
+      {/* Page Header */}
+      <Flex
+        backgroundColor="#F5F5F5"
+        marginY="12px"
+        textAlign="center"
+        height="42.5px"
+        alignItems="center"
+      >
+        <Heading fontSize="14px" fontWeight="700" color="#1A202C" flex="4.8">
+          Orders Details
+        </Heading>
+        <Heading fontSize="14px" fontWeight="700" color="#1A202C" flex="3.1">
+          Shipping Information
+        </Heading>
+        <Heading fontSize="14px" fontWeight="700" color="#1A202C" flex="1.0">
+          Total Paid
+        </Heading>
+        <Heading
+          fontSize="14px"
+          fontWeight="700"
+          color="#1A202C"
+          flex="1.0"
+          display={"flex"}
+          justifyContent="center"
+          alignItems="center"
+          gap="5px"
+        >
+          Status
+          <OrderStatusDropdownFilter handleValueChange={handleValueChange} />
+        </Heading>
+      </Flex>
+      {/* search orderID and user ID filter */}
+      {(filterdOrders || orders.data || [])
+        // order status filter
+        .filter((order: IOrder) => {
+          if (!filterStatus) return true;
+          return (
+            (filterStatus.isUnpaid && order.status == "unpaid") ||
+            (filterStatus.isProcessing &&
+              !(order.status == "unpaid") &&
+              !(order.status == "completed") &&
+              !(order.status == "cancelled")) ||
+            (filterStatus.isCompleted &&
+              !(order.status == "unpaid") &&
+              !(order.status == "processing") &&
+              !(order.status == "cancelled")) ||
+            (filterStatus.isCancelled &&
+              !(order.status == "unpaid") &&
+              !(order.status == "completed") &&
+              !(order.status == "processing"))
```

Comment:
can use "==="

Reply:
sure, will fix

Reply:
![image](https://user-images.githubusercontent.com/106716226/201942248-d7551234-7f12-482f-ad10-ef3f8892efd8.png)

Reply:
change logic to backend